### PR TITLE
initial setup

### PR DESCRIPTION
**Pull request by joepav**

### CODE REVIEW

**Review by Donaldblanc — APPROVED** [review]
later we can discuss the cleaning up of these comments and not have it feel as much as a template.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/jdcj-oslabs/apimocking/issues"
+  },
+  "homepage": "https://github.com/jdcj-oslabs/apimocking#readme",
+  "dependencies": {
+    "electron": "^5.0.2",
+    "save-dev": "^2.0.0"
+  }
+}
```
Comment (conor-jh) [inline package.json]: Let’s follow the lint rules here to have a new line at the end of each file

File: main.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+})
+
+app.on('activate', function () {
+  // On macOS it's common to re-create a window in the app when the
+  // dock icon is clicked and there are no other windows open.
+  if (mainWindow === null) createWindow()
+})
+
+// In this file you can include the rest of your app's specific main process
+// code. You can also put them in separate files and require them here.
```
Comment (conor-jh) [inline main.js:49]: Agree with @Donaldblanc that as we continue we should get rid of these comments, but it’s probably not necessary right now.

**Review by conor-jh — CHANGES_REQUESTED** [review]
Can you add a commit to include an `.eslintrc`?

**Review by conor-jh — APPROVED** [review]

**Pull request merged**